tutorials/containers/using-gitlab-as-a-container-registry/using-gitlab-as-a-container-registry.md

```diff
@@ -0,0 +1,244 @@
+---
+id: using-gitlab-as-a-container-registry
+summary: Learn how to use GitLab as a private container registry
+categories: containers
+tags: kubernetes, containers, gitlab, cicd
+difficulty: 3
+status: draft
+feedback_url: https://github.com/canonical-websites/tutorials.ubuntu.com/issues
+published: 2019-09-02
+author: Peter De Sousa <peter.de.sousa@canonical.com>
+---
+
+# Using GitLab as a container registry
+
+## Introduction
+
+The aim of this tutorial post is to demonstrate how you can use Charmed Kubernetes in a CI/CD environment using GitLab’s container registry functionality.
```

Comment:
Move it below the next paragraph and structure in a form of bullet points (you can reference some recent tutorials like https://tutorials.ubuntu.com/tutorial/install-microk8s-on-mac-os#0)